pkgs/applications/audio/zxtune/default.nix

```
@@ -0,0 +1,129 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, boost
+, zlib
+# File backends (for decoding and encoding)
+, withMp3 ? false
+, lame
+, withOgg ? false
+, libvorbis
+, withFlac ? false
+, flac
+# Audio backends (for playback)
+, withOpenal ? false
+, openal
+, withSDL ? !stdenv.hostPlatform.isLinux
+, SDL2
+, withOss ? false
+, withAlsa ? stdenv.hostPlatform.isLinux
+, alsa-lib
+, withPulse ? stdenv.hostPlatform.isLinux
+, libpulseaudio
+# GUI audio player
+, withQt ? true
+, qt5
+, zip
+}:
+let
+  dlopenBuildInputs = []
+    ++ lib.optional withMp3 lame
+    ++ lib.optional withOgg libvorbis
+    ++ lib.optional withFlac flac
+    ++ lib.optional withOpenal openal
+    ++ lib.optional withSDL SDL2
+    ++ lib.optional withAlsa alsa-lib
+    ++ lib.optional withPulse libpulseaudio;
+  supportWayland = (!stdenv.hostPlatform.isDarwin);
+  platformName = "linux";
+  staticBuildInputs = [ boost zlib ]
+    ++ lib.optional withQt (if (supportWayland) then qt5.qtwayland else qt5.qtbase);
+in with lib;
+stdenv.mkDerivation rec {
+  pname = "zxtune";
+  version = "r5054";
```

```suggestion
  version = "5054";
```
Usually the version attribute starts with a digit.

---

Done.